### PR TITLE
v0.1.4

### PR DESCRIPTION
**Pull request by emanspeaks**
Please create a release named `v0.1.4` after this is merged.  Subsequent changes will require a new release each time in order for pip to pick them up.

We will also need to update the URL used in `setup.py` files for packages using this.

### CODE REVIEW

File: fastnml/reader.py [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                    nml[key] = [nml[key], value]
+            else:
+                nml[key] = value
+
+    if not parser:
+        parser = Parser()
+
+    namelists = _split_namelist_file(filename)
+    results = list()
+    results_append = results.append
```
Comment (jacobwilliams) [inline fastnml/reader.py:229]: why do this?

File: fastnml/reader.py [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                        started = False
+                        namelists[i].append(line)
+                        continue
+                    if started:
+                        namelists[i].append(line)
+
+    return namelists
+
+
+def read_namelist(filename: str, *, n_threads: int = 4,
```
Comment (jacobwilliams) [inline fastnml/reader.py:203]: What is this `*` here for? I'm not familiar with this syntax?

Comment (jacobwilliams) [inline fastnml/reader.py:203]: I think `n_threads=0` should be the default right?

File: LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (jacobwilliams) [inline LICENSE.txt:19]: Are your OK with me changing this to the 3-clause BSD license?

File: tests/test.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+def read_chunks_simple(filename, n_threads, parser):
+    'fastnml: Read in chunks'
+    return read_namelist(filename, n_threads=n_threads, parser=parser)
+
+
+def run_read_tests(filenames, tests, parser, repeats):
+    # screensaver code from https://eddiejackson.net/wp/?p=23435
+    # disable screensaver
+    ctypes.windll.kernel32.SetThreadExecutionState(0x80000002)
```
Comment (jacobwilliams) [inline tests/test.py:30]: what is this?

File: tests/test.py [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+            for n_threads in range(0, threads + 1):
+                # case = f"{func.__doc__} ({n_threads} threads) : "
+                case = "{} ({} threads) : ".format(func.__doc__, n_threads)
+                tottime = timeit(lambda: func(filename, n_threads, parser),
+                                 number=repeats)
+                # print(f"{case.ljust(55)}{tottime/repeats} sec")
+                print("{}{} sec".format(case.ljust(55), tottime/repeats))
+
+    # enable screensaver
+    ctypes.windll.kernel32.SetThreadExecutionState(0x80000000)
```
Comment (jacobwilliams) [inline tests/test.py:55]: what is this?

File: fastnml/reader.py (hunk 1 above)
Comment (emanspeaks) [reply fastnml/reader.py:229]: setting `results_append = results.append`?  It provides a shortcut to the exact method on the object so it does not need to resolve the dot operator every iteration through the loop.  Provides a very slight increase in speed as a result.

**Comment by jacobwilliams** [comment]
changes merged via #3

**Pull request closed without merging**